packages/js/src/executors/swc/swc.impl.ts

```diff
@@ -38,15 +39,28 @@ function normalizeOptions(
     outputPath
   );
 
+  /**
+   * Reduce outputPath to root of layout dir (libsDir/appsDir)
+   * in desired build output dir (default is dist)
+   *
+   * dist/packages/lib-one -> dist/packages
+   * build/packages/lib-one -> build/packages
+   * dist/packages/lib-one/nested/nested-lib-one -> dist/packages
+   * dist/libs/lib-one -> dist/libs
+   * dist/apps/app-one -> dist/apps
+   */
+  const swcDestPath =
```

Comment:
This wouldn't work if the `outputPath` does not include  the layout dir.

Probably an  edge case, but you could set `outputPath: 'dist/utils'` for a package in `packages/utils`. In this case the destination would be `libs` instead of `packages/utils`.

Reply:
I found a workaround that is the least "guess-work". `outputPath` isn't "computed" anymore. Example:

```json
{
   utils: {
      projectRoot: 'packages/utils',
      targets: {
         executor: nrwl/js:swc,
         options: {
            outputPath: 'dist/utilities'
         }
      }
   }
}
```

With the above project config, swc will execute:

```shell
npx swc utils -d ../dist/utilities
```

The `outputPath` is kept intact which allows the consumers to fully customize it while making `swc` works correctly.